cli/mender-artifact/copy.go

```diff
@@ -143,6 +143,26 @@ func Install(c *cli.Context) (err error) {
 	}
 }
 
+func Remove(c *cli.Context) (err error) {
+	comp, err := artifact.NewCompressorFromId(c.GlobalString("compression"))
+	if err != nil {
+		return cli.NewExitError("compressor '"+c.GlobalString("compression")+"' is not supported: "+err.Error(), 1)
+	}
+	if c.NArg() != 1 {
+		return cli.NewExitError(fmt.Sprintf("Got %d arguments, wants one", c.NArg()), 1)
+	}
+	if !isimg.MatchString(c.Args().First()) {
+		fmt.Println(c.Args())
+		return cli.NewExitError("The input image does not seem to be a valid image", 1)
```

Comment:
This makes it sound like the contents are wrong. Suggest rephrasing to `The input image does not have a valid extension`.